### PR TITLE
Docker build for ARM (Raspberry Pi)

### PR DESCRIPTION
**Pull request by Alkarex**
Using QEMU for building the image on Docker Hub, then native execution.
Inspired by https://github.com/stas-demydiuk/domoticz-docker

### CODE REVIEW

**Comment by Alkarex** [comment]
Using QEMU for building the image on Docker Hub, then native execution.

**Comment by Alkarex** [comment]
Merging in /dev to test the full pipeline

**Comment by Alkarex** [comment]
Seems to work fine :-)
https://hub.docker.com/r/freshrss/freshrss/tags

**Comment by Alkarex** [comment]
The tag `freshrss/freshrss:dev-arm` is already available for testing

File: Docker/hooks/pre_build [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ "$DOCKERFILE_PATH" == *-ARM ]];
+then
```
Comment (SuperSandro2000) [inline Docker/hooks/pre_build:4]: nitpicky: you usually write this on the same line with if and you don't need to quote vars when using test.

Comment (Frenzie) [reply Docker/hooks/pre_build:4]: @Alkarex PR on my end incoming soon.

**Pull request merged**